validation-script/bsmLogDuringEvent.ini

```diff
@@ -0,0 +1,132 @@
+[_settings]
+KafkaTopic = topic.OdeBsmJson
+InputFilePath = ../data/bsmLogDuringEvent.gz
+ExpectedMessages = 222
+
+[bsmSource]
+Path = metadata.bsmSource
+Type = enum
+Values = ["RV", "EV", "unknown"]
+
+[payloadType]
+Path = metadata.payloadType
+Type = string
+EqualsValue = us.dot.its.jpo.ode.model.OdeBsmPayload
```

Comment:
I was not sure if the EqualsValue need to be assigned to a string.  In the Excel spreadsheet does not have a default value.